src/utils/types/zetkin.ts

```diff
@@ -172,6 +172,9 @@ export interface ZetkinSurvey {
   allow_anonymous: boolean;
   access: string;
   callers_only: boolean;
+  published: string | null;
+  expires: string | null;
+  elements: ZetkinSurveyElement[];
```

Comment:
doesn't it need to be in alphabetical order?

Reply:
we don't have linting rules for alphabetical order inside types/interfaces. When I create a type myself I usually put them in alpahbetical order, but since this was just adding to an existing type I put them at the bottom.